gem/templates/profiles/register.html

```diff
@@ -19,6 +19,10 @@
             <p><strong>{% trans "Choose a 4-digit pin" %}</strong></p>
             {{ form.password.errors}}
             {{ form.password }}
+
+            <p><strong>{% trans "What gender are you" %}</strong></p>
```

Comment:
We probably want `Choose your gender` to keep it the same as the other questions.